app/Http/Controllers/Auth/AuthController.php

```diff
@@ -2,11 +2,11 @@
 
 namespace Rogue\Http\Controllers\Auth;
 
-use Rogue\User;
-use Validator;
 use Rogue\Http\Controllers\Controller;
 use Illuminate\Foundation\Auth\ThrottlesLogins;
 use Illuminate\Foundation\Auth\AuthenticatesAndRegistersUsers;
```

Comment:
You can remove these two traits (`ThrottlesLogins`, `AuthenticatesAndRegistersUsers`, and the corresponding `use` statements below in the class). They don't do anything with the Northstar login methods.